up42/processing.py

```diff
@@ -131,7 +131,7 @@ def update():
             self.collection_url = job.collection_url
             self.errors = job.errors
             self.credits = job.credits
-            if self.status not in [JobStatus.CAPTURED, JobStatus.RELEASED]:
+            if self.status not in [JobStatus.CAPTURED, JobStatus.RELEASED, JobStatus.INVALID, JobStatus.REJECTED]:
```

Comment:
Should we also check for the `unlicensed` state? Even though it should not be possible to have it when starting a job from the SDK?

Reply:
Added